tests/test0183.d

```diff
@@ -0,0 +1,10 @@
+//T compiles:yes
+//T has-passed:yes
+//T retval:42
+
+int main() {
+	int a = mixin("-12");
```

Comment:
I'd like to test feature combinations in a separate test...
this one is supposed to unit-test mixed-in-expressions.